src/promotion/ops.h

```diff
@@ -27,6 +27,9 @@ struct promotion_policy_ops {
 	void (*setup)(ocf_cache_t cache);
 		/*!< initialize promotion policy default config */
 
+	uint64_t (*size_of)(ocf_cache_t cache);
```

Comment:
I have mixed feelings about making it a part of promotion policy ops. For now it seems to be problem of nhit only, so maybe it would be enough to print message from `init()` and then return appropriate error code.

Reply:
The extended API was meant to provide some fallback options, but it's changed since that so maybe it's not needed